test/e2e/snaps/ppom-toggle-settings.spec.js

```diff
@@ -0,0 +1,87 @@
+const { strict: assert } = require('assert');
+const {
+  withFixtures,
+  unlockWallet,
+  openDapp,
+  defaultGanacheOptions,
+  getWindowHandles,
+} = require('../helpers');
+const FixtureBuilder = require('../fixture-builder');
+
+const switchToEthereumMainnet = {
```

Comment:
nit: name could be a noun, as the variable only holds data, something like `mainnetProviderConfig`, instead of a verb

Reply:
Updated here. https://github.com/MetaMask/metamask-extension/pull/20454/commits/0a0caa3873547577376b0f63a4cb8c3eb4ff04f1